docs/documentation/api-specification/database-api.md

```diff
@@ -54,11 +54,25 @@ and delete operation on this locked record.
 ## Features
 
 ### Create a record
-To create a record TODO 
+The example below illustrates how to create and set a record.
+
+Example:
+```groovy
+public void main() {
```

Comment:
This is not the complete code for inserting a record in the database. Below is an example of how it is done.

```groovy
DBAction query = database.table("MITMAS")
      .index("00")
      .selection("MMCONO", "MMITNO", "MMITDS", "MMSTAT")
      .build();
DBContainer container = query.getContainer()
container.set("MMITDS", "Sample Item")
query.insert(container)
// Or if it's needed to update the already existing record
query.insert(container, { LockedResult existingRecord ->
  existingRecord.set("MMITDS", "Updated description")
  existingRecord.update()
})
```